### ISSUE TITLE
[BUG] Missing Quest Tracker background

### ISSUE THREAD

**Issue by n1xx1** [League-of-Foundry-Developers/foundryvtt-forien-quest-log#14]
**Module Version:** v0.7.9

**Describe the bug**
The background of the quest tracker isn't loaded when running foundry in a subdirectory.

**To Reproduce**
1. Install this docker image: https://hub.docker.com/r/felddy/foundryvtt
2. Set FOUNDRY_ROUTE_PREFIX to something like '/foundry'
3. Open the quest tracker.

**Expected behavior**
I expect the quest tracker to have the correct background.

**Browser:**
Any.

**Foundry Version:** 
Doesn't matter.

**Game System:** 
Doesn't matter.

**Additional context**
The path of the background should be `../../../ui/denim075.png` instead of `/ui/denim075.png`.

**Comment by typhonrt** [comment]
@n1xx1 Thanks for reporting this issue. While slightly an inconvenience I recommend modifying the quest log styles by hand since you have the temporary fix and can set the relative path. I'll certainly keep this in mind if there is another maintenance release of FQL which there very well might be in the coming month or two for the v10 release of Foundry.

**Comment by typhonrt** [comment]
Hi @mowngle. If you are modifying the installed version of FQL then you need to modify the final CSS file located in `/Data/modules/forien-quest-log/css/init.css`

Looks like the denim image reference is right at the front of that file.

**Comment by n1xx1** [comment]
My solution was to install the [Custom CSS module](https://foundryvtt.com/packages/custom-css) and use this bit of css:

```
#quest-tracker.fql-app {
    background: url(./ui/denim075.png) repeat !important;
    background: var(--palette-fql-qt-image-background,url(./ui/denim075.png)) repeat !important;
}
.fql-app {
    background: url(./ui/denim075.png) repeat !important;
}
```

Hope it helps, @mowngle

**Comment by typhonrt** [comment]
Good news folks... This is fixed pending release.  There will be a FQL 0.7.10 release just after Foundry core v10 reaches "stable". This release works on v9 & v10 and includes several other quality of life updates that will work on either Foundry version. 

I'll close this issue after FQL 0.7.10 releases soon.

**Comment by typhonrt** [comment]
Hi @n1xx1 and @mowngle. This issue is fixed in the public release of FQL v0.7.10 that is available now for Foundry v9 & v10.
